docs/proposals/mapper-support-streaming-data.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Mapper Support Streaming Data
```

Comment:
The proposal is mainly focus on the `video stream data`, and stream data should include many other types, so forr the title, how about change to `Mapper Support Video Streaming Data`?  cc @WillardHu @wbc6080 HDYT

Reply:
Done @fisherxu